core-web/libs/edit-content/src/lib/feature/edit-content/store/features/content.feature.ts

```diff
@@ -233,12 +234,12 @@ export function withContent() {
                  * @returns {Observable<string>} An observable that emits the content's inode when initialization is complete
                  * @throws Will redirect to /c/content and show error if initialization fails
                  */
-                initializeExistingContent: rxMethod<string>(
+                initializeExistingContent: rxMethod<{ inode: string; depth: DotContentletDepth }>(
```

Comment:
What about setting the depth of 2 by default, and not touching the actual code?

Reply:
I think, for the future, we need to change the depth for certain cases. Therefore, I believe it should be passed dynamically